src/ExExpression.elm

```diff
@@ -475,10 +491,14 @@ operators =
     , ( "/", "/" )
     , ( ">>", ">>>" )
     , ( "<|", "<<<" )
-    , ( "%", "" ) -- Exception
-    , ( "//", "" ) -- Exception
-    , ( "rem", "" ) -- Exception
-    , ( "^", "" ) -- Exception
+    , ( "%", "" )
+      -- Exception
+    , ( "//", "" )
+      -- Exception
+    , ( "rem", "" )
+      -- Exception
+    , ( "^", "" )
+      -- Exception
```

Comment:
We probably have different `elm-format` version. We can set [lint-staged](https://github.com/okonet/lint-staged) for that so formatting will be consistent.